### DISCUSSION TITLE
一直重定向，无法申请学生包

### DISCUSSION THREAD

**Discussion by kroblus** [community/community#76954] [GitHub Education]
### Hi everyone,

打开学生包认证地址：https://education.github.com/discount_requests/application会自动重定向到https://github.com/pricing，无法完成学生认证

**Comment by xiaoxiaojiujiu** [comment]
我也一样，一直跳转到price，无法认证、查看申请资格

**Reply by Log4J-Ops** [reply]
do you have some solutions?

**Comment by AdeCaspeer** [comment]
有解决吗
我也是这样的问题

**Comment by liuliangsir** [comment]
Same issue here

https://github.com/community/community/assets/8049878/ea8a944d-aab8-4074-90af-f565d51c65f4

**Comment by jiajiaxd** [comment]
same issue

**Comment by Log4J-Ops** [comment]
same issue，it makes me crazy

**Comment by lanbinshijie** [comment]
same issue as me, is that a restriction of region?

**Comment by FeiFei-SS** [comment]
Same too..

**Comment by ghostinhershell** [comment]
Hi @kroblus, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
